airflow/www/api/experimental/endpoints.py

```diff
@@ -150,23 +284,39 @@ def get_dag_code(dag_id):
         return response
 
 
+@api_experimental.route('/dags/<string:dag_id>/tasks', methods=['GET'])
+@requires_authentication
+def tasks(dag_id):
+    """Returns a JSON with all tasks associated with the dag_id. """
+    try:
+        task_list = list()
+        task_ids = get_tasks(dag_id)
+        for task_id in task_ids:
+            task_list.append(get_task_as_dict(dag_id, task_id))
+
+    except AirflowException as err:
+        _log.info(err)
+        response = jsonify(error="{}".format(err))
+        response.status_code = err.status_code
+        return response
+
+    return jsonify(task_list)
```

Comment:
Good practice to initialize task_list = list() just before the try-except since the return statement depends on it.

Reply:
where is info? but I have indeed moved the `task_list = list()` statement above the try block.

Reply:
This was a copy/paste of the same thing, please initialize ```task_list = list()``` just before the "try"